python/hail/linalg/matrix.py

```diff
@@ -108,5 +110,21 @@ def persist(self, storage_level='MEMORY_AND_DISK'):
     def unpersist(self):
         return BlockMatrix(self._jbm.unpersist())
 
-block_matrix_type.set(BlockMatrix)
+    @handle_py4j
+    def to_local_matrix(self):
```

Comment:
I think this should be to_numpy_matrix. I want to reserve to_local_matrix for converting a hail matrix from distributed to local.